### PR TITLE
[RECEIVING] .on_destroy - custom destroying logic

### PR DESCRIPTION
**Pull request by 0exp**
- See `CHANGELOG.md` and `docs/synomsis.md`

### CODE REVIEW

File: docs/synopsis.md [hunk 1]
```diff
@@ -203,6 +203,10 @@ This method implements logic of mapping `source` to `to_table` and allows custom
 You can use one `source` for a lot of `to_table`.
 
 The following options are available inside the block:
+- `on_destroy` - defines a custom logic and behavior for `destroy` event:
+  - definition: `on_destroy { |attributes:|, |target_keys:| ...your code... }`
```
Comment (tycooon) [inline docs/synopsis.md]:
```ruby
on_destroy { |attributes:|, |target_keys:| ...your code... }
```
This doesn't look like a valid ruby code to me :(

File: spec/features/receiving_handler_spec.rb [hunk 2]
```diff
@@ -465,6 +465,50 @@ def fire_destroy_event
         end
       end
 
+      context "when on_destroy  is defined" do
```
Comment (tycooon) [inline spec/features/receiving_handler_spec.rb]: Let's remove the extra space.

File: docs/synopsis.md (hunk 1 above)
Comment (0exp) [reply docs/synopsis.md]: oops :)

File: spec/features/receiving_handler_spec.rb (hunk 2 above)
Comment (0exp) [reply spec/features/receiving_handler_spec.rb]: macbook keyboard double-click bug ;(

**Pull request merged**